tm2py/components/network/active/active_modes.py

```diff
@@ -350,49 +350,38 @@ def _export_results(
         )
         # convert node id to sequential (1-based) zone id
         # consistent with tm2.1 - java expects this
-        zone_seq_file = self.get_abs_path(self.controller.config.scenario.zone_seq_file)
-        zone_seq_df = pd.read_csv(zone_seq_file)
+        zone_seq_df = self.controller.node_seq_id_xwalk
         taz_seq = dict(
             zip(
-                zone_seq_df[zone_seq_df.TAZSEQ > 0].N,
+                zone_seq_df[zone_seq_df.TAZSEQ > 0].model_node_id,
                 zone_seq_df[zone_seq_df.TAZSEQ > 0].TAZSEQ,
```

Comment:
Could we be consistent with our convention and include the underscore: `TAZ_SEQ`?